apps/docs/src/app/core/component-docs/card/examples/list-card/card-list-example.component.html

```diff
@@ -0,0 +1,49 @@
+<fd-card cardType="object" class="docs-flex-item-margin">
+    <fd-card-header>
```

Comment:
Could you add `[interactive]="false"` there? card with list shouldn't have hoverable/focusable header

Reply:
fixed.